src/components/Blocks_Components/ManualBlocks.js

```diff
@@ -1,13 +1,16 @@
-import React, { useState } from "react";
-import { connect } from "react-redux";
-import { selectCategories } from "../../redux/actions/ManualActions";
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import {
+	selectCategories,
+} from '../../redux/actions/ManualActions';
```

Comment:
Formatting is weird here, can we run prettier on all the files and recommit them?